src/gql/queries/get-mainline-commits.graphql

```diff
@@ -0,0 +1,28 @@
+query MainlineCommits($options: MainlineCommitsOptions!) {
+    mainlineCommits(options: $options) {
+        versions {
+            version {
+            id
```

Comment:
nit: can we fix some of the indentation in this file. this plugin that I've used before may help: https://marketplace.visualstudio.com/items?itemName=GraphQL.vscode-graphql